modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java

```diff
@@ -80,12 +80,12 @@ public class ServiceOrderEntity {
     @Convert(converter = ObjectJsonConverter.class)
     private List<DeployResource> previousDeployedResources;
 
-    @Column(name = "PREVIOUS_DEPLOYED_RESULT_PROPERTY", columnDefinition = "json")
+    @Column(name = "PREVIOUS_DEPLOYED_SERVICE_PROPERTY", columnDefinition = "json")
```

Comment:
can you also please check the lines 68 and 73. there is a typo there with the column name. If possible, we can fix with the same PR.

Reply:
Updated.